contents/handbook/engineering/support-hero.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Support Hero
+sidebar: Handbook
+showTitle: true
+---
+
+Every release cycle, we assign one person to be the "Support Hero." Their primary job is to help customers, debug/fix issues, and debug/fix sentry errors, in that order of priority. All other work takes a back seat while they doing support.
+
+There are a couple of channels that customer requests come in so make sure you keep an eye on all of them:
+
+- [PostHog Users's Slack](https://posthog.com/slack)
+  - Make sure you're in #general, #getting-started and #introductions (as sometimes people will introduce themselves and ask a question at the same time)
+  - Also make sure you're in the private #_customer_support channel where Papercups conversations get routed
+- [HubSpot inbox](https://app.hubspot.com/live-messages/6958578/inbox/)
+- GitHub issues, with [the main repo](https://github.com/posthog/posthog/issues) being the most important one
+
+## Tips
+
```

Comment:
also, ctrl+f'ing the slack users channel occasionally works because questions are repeated